### ISSUE TITLE
Overlapping Text In Debug Build 

### ISSUE THREAD

**Issue by akano12** [libre-tube/LibreTube#4123]
### Steps to reproduce

1. Have no internet connection (connected to wifi/mobile data but there's no connection)
2. Open LibreTube
3. Refresh to reload content whne connections is restored 

### Expected behavior

1. Load data from cache or display a circular progress indicator when there's no connection
2. Reload content properly after connection is restored. 

### Actual behavior

When there's no connection, this is displayed
![Screenshot_20230627_011823_LibreTube Debug](https://github.com/libre-tube/LibreTube/assets/50032603/568bf34c-3269-4010-b073-63a0442d70b7)

When connection is restored, it will overlap the actual content
![Screenshot_20230627_011148_LibreTube Debug](https://github.com/libre-tube/LibreTube/assets/50032603/7a2bac7a-9329-46fd-aecf-0ba7ed3c8e5b)

### LibreTube version

Latest Debug 

### Android version

Android 13

### Other details

On stable 0.15, there's a circular progress indicator if there's no connection.
On debug there's this text that says "Nothing Here". 
Tl;dr, works properly on stable 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Stable doesn't work properly because it loads forever when you have no internet connection or you unchecked all options to display in the home tab.
